jumpscale/packages/admin/actors/wallet.py

```diff
@@ -19,15 +19,11 @@ def create_wallet(self, name: str, wallettype: str = None) -> str:
         wallet = j.clients.stellar.new(name=name, network=wallettype)
 
         try:
-            if wallettype == "TEST":
-                wallet.activate_through_friendbot()
-            else:
-                wallet.activate_through_threefold_service()
+            wallet.activate_through_threefold_service()
```

Comment:
activating thought testnet using threefold service activates but gives me that error, is that related ?


```
TypeError: string indices must be integers
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/waleed/js-sdk/jumpscale/clients/stellar/stellar.py", line 246, in activate_through_threefold_service
    self._activation_account(activationdata["activation_code"])
    │                        └ '{"activation_code": "abcd", "address": "GAJ4FG4DA2M3BUECSANIRX3NWVVHICUBMS2Y24Z4ECLCMZFZCDE6T3YM", "phonenumbers": ["+123456789...
    └ Stellar(
  instance_name='ell',
  network=<Network.TEST: 'TEST'>,
  address='GAJ4FG4DA2M3BUECSANIRX3NWVVHICUBMS2Y24Z4ECLCMZFZCDE...
TypeError: string indices must be integers

string indices must be integers
JS-NG> wallet.get_balance()                                                                                                                                                                                   
Balances
  3.6000000 XLM
```

Reply:
The activation service was rewritten  but this is exactly why testnet should not use a different setup than mainnet, the activation service on testnet is not tested at the moment. Good thing you found this, will check what is going wrong

Reply:
Found it, there is an avtivateaccount call that does nothing and should not be called, It used to return no error but it does in the rewrite. Will fix it.

Reply:
Done, added the commit to this PR

Reply:
Apparently no-one noticed the error before even though it's a client error and is in there since april or so

Reply:
thanks